### PR TITLE
831668 -UG Documentation for MAUI Expander Appearance

### PR DESCRIPTION
**Pull request by AnanthaLakshmiKannan**
Documentation done for MAUI Expander Appearance

### CODE REVIEW

File: MAUI/Expander/appearance.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+title: Appearance in .NET MAUI Expander control | Syncfusion
+description: Learn here all about Appearance support in Syncfusion .NET MAUI Expander (SfExpander) control and more.
+platform: MAUI
+control: SfExpander
+documentation: ug
+---
+
+# Appearance in .NET MAUI Expander (SfExpander)
+
+The Expander allows to customize the appearance of the Icon and provides various functionalities to the users.
```
Comment (PiruthivirajM) [inline MAUI/Expander/appearance.md:12]: The .NET MAUI SfExpander comes with built-in support for customizing the appearance of the header.

**Review by PiruthivirajM — COMMENTED** [review]
Address review comments.

File: MAUI/Expander/appearance.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+The appearance of the `SfExpander` can be customized using the following two `VisualStates`:
+
+* Expanded
+* Collapsed
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfExpander x:Name="expander1" IsExpanded="True" >
+    <syncfusion:SfExpander.Header>
```
[the remark this answers is not in the document]
Comment (AnanthaLakshmiKannan) [reply MAUI/Expander/appearance.md:65]: Addressed the concerns provided.

File: MAUI/Expander/appearance.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+commonStateGroup.States.Add(expanded);
+commonStateGroup.States.Add(collapsed);
+
+visualStateGroupList.Add(commonStateGroup);
+VisualStateManager.SetVisualStateGroups(expander, visualStateGroupList);
+this.Content = expander;
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Expander with VSM](Images/appearance/maui-expander-with-vsm.png)
```
[the remark this answers is not in the document]
Comment (AnanthaLakshmiKannan) [reply MAUI/Expander/appearance.md]: Addressed the concerns provided.

**Review by PiruthivirajM — APPROVED** [review]
Approved

**Pull request merged**